ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/EditorSteps.java

```diff
@@ -159,9 +161,12 @@ public void checkIfWarningIsVisibleInRange(int start, int finish) {
 
     @When("^open integration flow details")
     public void openIntegrationFlowDetails() {
-        if (editor.getRootElement().$(Element.HIDDEN_DETAILED_VIEW).exists()) {
+        try {
+            OpenShiftWaitUtils.waitFor(() -> editor.getRootElement().$(Element.HIDDEN_DETAILED_VIEW).exists(), 15 * 1000);
             log.info("Expander is closed, opening details");
             editor.getRootElement().$(Element.EXPANDER).shouldBe(visible).click();
+        } catch (InterruptedException | TimeoutException e) {
+            log.warn("Expander to open was not found!");
```

Comment:
wouldn't this cause a failure?

Reply:
no because sometimes it is already opened so I want to continue the test :)